code/game/objects/items/devices/instruments.dm

```diff
@@ -43,16 +43,16 @@
 	song.interact(user)
 
 /obj/item/instrument/violin
-	name = "space violin"
-	desc = "A wooden musical instrument with four strings and a bow. \"The devil went down to space, he was looking for an assistant to grief.\""
+	name = "violin"
+	desc = "A wooden musical instrument with four strings and a bow. ""
```

Comment:
has an extra space and two quotation marks
You can also just change the reference to the original lyrics.